src/server/components/lists/controllers/ingest/ingestPutController.ts

```diff
@@ -80,6 +82,8 @@ export async function ingestPutController(req: Request, res: Response) {
       },
     };
 
+    await sendProviderChangeDetailsEmailToAdmins(listItem.list as Partial<List>, listItem.status);
```

Comment:
This should be sent closer to where `sendAnnualReviewCompletedEmailForList` is sent. 

If for whatever reason prisma.listItem.update fails, it means post will get an email but won't see an update.

Reply:
Not quite sure what you mean by the second sentence. If you're referring to Line **41**
```js
 const listItem = await prisma.listItem.findUnique({
  // ...
 ```

It looks like there is some code to check if it is null.